mslice/plotting/plot_window/plot_figure_manager.py

```diff
@@ -87,14 +85,14 @@ def flag_as_current(self):
 
     def add_slice_plot(self, slice_plotter_presenter, workspace):
         if self._plot_handler is None:
-            self.move_window(-self.window.width() / 2, 0)
+            self.move_window(self.window.width() * 1.05, self.window.height() / 2)
```

Comment:
Agreed that this is a fine thing to do. Could we add a code comment about why we are doing it? At first glance it appears a little strange.